includes/payment-methods/class-wc-stripe-upe-payment-gateway.php

```
@@ -4236,4 +4244,67 @@ private function maybe_add_presentment_metadata_to_order( WC_Order $order ): voi
+		$checkout_session_id = $order_helper->get_stripe_checkout_session_id( $order );
+		if ( ! $checkout_session_id ) {
+			return null;
+		}
+
+		$this->maybe_add_presentment_metadata_to_order( $order );
+
+		return [
+			'amount'   => (int) $order_helper->get_stripe_presentment_amount( $order ),
+			'currency' => $order_helper->get_stripe_presentment_currency( $order ),
+		];
+	}
```

PHPMD (2.15.0)

[error] The method get_presentment_data_from_order is not named in camelCase. (undefined)

(CamelCaseMethodName)